package.json

```diff
@@ -11,7 +11,7 @@
     "test": "mocha",
     "cover": "rm -Rf coverage/* && istanbul cover _mocha -- -R spec",
     "lint": "eslint bin/* lib/*.js lib/**/*.js test/*.js",
-    "install": "cd pkcs7 && bundle install --path='./'",
+    "install": "cd pkcs7 && bundle package",
```

Comment:
would it make sense to remove .bundle dir from the .gitignore file? That way everyone would use same configs.